Makefile

```diff
@@ -1,8 +1,11 @@
 BIND_ADDR ?= :8081
 BINPATH ?= build
 
+BUILD_TIME=$(shell date +%s)
+GIT_COMMIT=$(shell git rev-parse HEAD)
 VERSION=`git describe --tags`
```

Comment:
We do not need this variable, we should use `HC_VERSION` but rename to `VERSION`